Apps.Lexeri/Webhooks/WebhookList.cs

```diff
@@ -0,0 +1,60 @@
+using System.Net;
+using Blackbird.Applications.Sdk.Common.Webhooks;
+using Newtonsoft.Json;
+using Apps.Lexeri.Models.Response;
+using Apps.Lexeri.Webhooks.Handlers;
+
+namespace Apps.Lexeri.Webhooks;
+
+[WebhookList]
+public class WebhookList
```

Comment:
For all webhooks. Also add an optional input value (most likely called "Identifier" for you (I do recommend to switch to ID)) to filter by the identifier. That way it will be more easily compatible with checkpoints.

Reply:
Ok, if I understand you correctly, I've added the optional ID input value here (and switched to using ID instead of identifier): https://github.com/bb-io/Lexeri/pull/1/commits/da626b15e9e98293bf092115039a445bf8133d4d